backend/src/security/security.py

```diff
@@ -48,14 +51,53 @@ def create_access_token(
     return encoded_jwt
 
 
+def get_password_hash(password):
+    """This function takes the plain password and makes a hash from it using CryptContext"""
```

Comment:
For future reference, you don't have to mention "This function" since this documentation is already for a function.  I do the same thing too and miss it frequently.  This can be simplified down to.
```suggestion
    """Takes the plain password and makes a hash from it using CryptContext"""
```